### PR TITLE
docs: Update restore docs to include new rollback feature

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

## What does this pull request change?
Updates backup / restore documentation in accordance with new features implemented

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-1497

## Specify the version of the product this pull request applies to
- 7.38.0

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [x] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [x] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/partials/assembly_backup-and-disaster-recovery.adoc [hunk 1]
```diff
@@ -21,6 +21,13 @@
 * Backup snapshots are bound to their own specific cluster and must be used only there.
 ====
 
+{prod} Operator creates a new backup on every {prod-short} update.
+Depending on the xref:define-backup-server-for-operator.adoc[configuration], backups before {prod-short} update are stored in:
+
+* The only configured backup server
+* Backup server which configuration marked with `che.eclipse.org/backup-before-update: true` annotation if there is more than one configuration
+* Internal backup server if no backup servers configured or configured several and `che.eclipse.org/backup-before-update: true` annotation missing
+
```
Comment (tolusha) [inline modules/administration-guide/partials/assembly_backup-and-disaster-recovery.adoc]: List means that backup is stored in an every item, but we have to mention a default configuration and how to change it.

File: modules/administration-guide/partials/proc_managing-backups-through-cli.adoc [hunk 2]
```diff
@@ -34,18 +34,43 @@ $ {prod-cli} server:backup --repository-url=__<repository-url>__  --repository-p
...
+. Recover to the previous state of {prod-short}:
 +
 [source,shell,subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:restore --repository-url=__<repository-url>__ --repository-password=__<repository-password>__
+$ {prod-cli} server:restore --repository-url=__<repository-url>__ --repository-password=__<repository-password>__ --snapshot-id=__<snapshot-id>__
 ----
 
-* The `server:restore` command can be run with no arguments when the backup server was used before.
+* It is possible to omit `--repository-url` and `--repository-password` arguments for the `server:restore` command if the same backup server should be used.
```
Comment (tolusha) [inline modules/administration-guide/partials/proc_managing-backups-through-cli.adoc]: I would remove this, it is recommended to provide sever configuration

**Review by tolusha — APPROVED** [review]

**Comment by mmorhun** [comment]
@themr0c @MichalMaler @rkratky please review someone.

**Comment by mmorhun** [comment]
I think we should wait for or merge into https://github.com/eclipse/che-docs/pull/2121

**Comment by MichalMaler** [comment]
My review this PR by @mmorhun is here.
https://docs.google.com/document/d/1J4PH7CRjWu_bSmzfP-Up09_eJ_XR3CQzXya00dm4SGc/edit

I have also a PR review, but i wanted to synchronize all names (of files, IDs, and headings) and I am not done yet.
https://github.com/eclipse/che-docs/pull/2133/files 

It is possible it will colide with what @max-cx is doing in his own PR.

**Review by MichalMaler — APPROVED** [review]

**Pull request merged**